lib/ProductOpener/Import.pm

```diff
@@ -1906,6 +1906,9 @@ sub import_csv_file($) {
 									require LWP::UserAgent;
 
 									my $ua = LWP::UserAgent->new(timeout => 10);
+									
+									# Some platforms such as CloudFlare block the default LWP user agent.
+									$ua->agent("Open Food Facts (https://world.openfoodfacts.org)");
```

Comment:
`$server_domain` and `lang('site_name')` could be  used here to make it more dynamic

Reply:
Good idea, I'll do that.